From linux-usb-devel-admin@lists.sourceforge.net Sun Oct  9 17:31:35 2005
From: Ben Dooks <ben-linux@fluff.org>
Message-ID: <20051010002350.GA27420@home.fluff.org>
Content-Disposition: inline
Subject: USB: S3C2410 OHCI - add driver owner field
Date: Mon, 10 Oct 2005 01:23:50 +0100

Initialise the .owner field of the driver with
the module that owns it, to aid in linking
drivers to modules.

Signed-off-by: Ben Dooks <ben-linux@fluff.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/host/ohci-s3c2410.c |    1 +
 1 file changed, 1 insertion(+)

--- gregkh-2.6.orig/drivers/usb/host/ohci-s3c2410.c
+++ gregkh-2.6/drivers/usb/host/ohci-s3c2410.c
@@ -474,6 +474,7 @@ static int ohci_hcd_s3c2410_drv_remove(s
 
 static struct device_driver ohci_hcd_s3c2410_driver = {
 	.name		= "s3c2410-ohci",
+	.owner		= THIS_MODULE,
 	.bus		= &platform_bus_type,
 	.probe		= ohci_hcd_s3c2410_drv_probe,
 	.remove		= ohci_hcd_s3c2410_drv_remove,
